omnibot/processor.py

```diff
@@ -44,7 +44,7 @@ def process_event(event):
     if event_type == "message" or event_type == "app_mention":
         try:
             with statsd.timer("process_event"):
-                logger.debug(
+                logger.info(
```

Comment:
Will this not potentially log the message itself? This is set to debug because it's sensitive and should only be output if the app is in debug mode (like when locally developing).